### PR TITLE
completes step 5

### PR DESCRIPTION
**Pull request by jcainuk**
Completes step 5 of NC-News

### CODE REVIEW

**Comment by jcainuk** [comment]
I think I may have accidentally deleted my tests - I'll check what I did here

File: app.test.js [hunk 1]
```diff
@@ -54,6 +54,45 @@ describe("/api/topics", () => {
...
+        });
+    });
+    test("200: each article object should have the correct properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+
+          expect(articles.length).toBe(13);
```
Comment (johnhutcheon) [inline app.test.js:79]: Nice testing, always good to see a length check before a forEach 👍

File: app.test.js [hunk 2]
```diff
@@ -54,6 +54,45 @@ describe("/api/topics", () => {
...
 
+describe("/api/articles", () => {
+  describe("GET articles default", () => {
+    test("200: responds with an array appropriate articles sorted in descending date order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then((response) => {
+          const { articles } = response.body;
+          console.log(articles, "<< inside the test");
```
Comment (johnhutcheon) [inline app.test.js:65]: Make sure you remove any console logs and comments before submitting 🙂

**Review by johnhutcheon — APPROVED** [review]
Nice work Jonathan, good to merge! 👍

**Pull request merged**